components/DrawControls.svelte

```diff
@@ -150,18 +151,22 @@
 
     // Highlight something in the sidebar when we hover on a feature in the map
     $map.on("mousemove", (e) => {
-      var newHoverEntry = null;
-      if (routeSnapper && !routeSnapper.isActive()) {
-        // TODO This whines about a layer missing, and I can't suppress with try/catch
-        const ids = drawControls.getFeatureIdsAt(e.point);
-        if (ids.length > 0) {
-          newHoverEntry = ids[0];
+      if ($currentlyEditing == null) {
```

Comment:
Preview of the logic to come. We'll have explicit and exclusive modes (select + edit the form, edit geometry, new point, new area, new route), and each one will restrict the interactions allowed. This is a small example of that kind of thing